pkg/account/account.go

```diff
@@ -24,14 +24,10 @@ import (
 var (
 	//go:embed version.txt
 	libraryVersion string
-	// UserAgent is sent in HTTP requests.
-	//
-	// The default value is "<main package name>/<commit hash> tesla-sdk/<version>".
-	UserAgent = buildUserAgent()
 )
 
-func buildUserAgent() string {
-	library := strings.TrimSpace("tesla-sdk/" + libraryVersion)
```

Comment:
The TrimSpace is to take out a newline at the end of version.txt. We either need to remove the newline (and strip it out in the Makefile build command) or add the TrimSpace back. The newline results in `net/http: invalid header field value for "User-Agent"`.